test/PoseidonGoldenTests/GoldenTestData/forge/ForgePac21/ForgePac21.vcf

```diff
@@ -0,0 +1,24 @@
+##fileformat=VCFv4.2
+##source=trident_v1.7.0.0
```

Comment:
Having this line in the golden test is a bit of a headdache. Will need to change with every version change...

Reply:
Yes, I was seriously considering not printing the version because of that. But I think it's actually nice to have it, in principle. Maybe I need to remove this line then from the outputing VCF file...

Reply:
OK, I've looked into our Golden test and it's actually pretty painful to make any manual changes to the files before they're written out. So I think either we want to keep the feature that trident outputs its own version into the VCF, then I guess we have to live with the fact that the Golden-test VCF files will change with every version, or we remove that feature from the VCF output. I can live with both, honestly. What do you think @nevrome ?

Reply:
I think it's good to have the trident version in there. Could be super helpful with debugging. So let's just accept this minor nuisance for now.

Maybe one day we can find a way to avoid this issue in the golden tests. Some chronicle/timetravel-related files also change all the time, so we already have a TODO there anyway.